MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.md
+include contrib/ **
```

Comment:
I'm not familiar with the DSL used in `MANIFEST.in`.

1. What does this do?
1. Why is it necessary?